docs/photoniq/fingerprinting/index.md

```diff
@@ -21,28 +21,86 @@ When a new user visits your site, Digital Fingerprinting will:
+
+### Bot Detection
+
+The process of identifying automated activity on a website is known as bot detection. This technique distinguishes between human and bot traffic on a website. Bot detection helps maintain a website's security and integrity by preventing spam, fraud, and other malicious activities. It analyzes a variety of browser attributes and gives a bot detection signal with one of three values:
+
+- Good bot - An example of a good bot is a bot belonging to a well-known search engine (Google, Amazon, Bing). Or a bot that monitors uptime. These are non-harmful bots.
+- Bad bot - Some bots connect to networks using IP addresses that are widely known and have been marked as banned on the internet. This ban list combines well-known industry-curated IP ban lists and our ban list.
+- Unknown bot - usually an automation tool like Selenium, Puppeteer, Playwright, or anything that pretends to be a good search bot.
+
+## Fingerprint Metrics
+
+When a visitor accesses your website, the Fingerprint service uniquely identifies them. At the same time, the fingerprint service collects metrics about the individual visitor. These metrics include device types (e.g., desktop, mobile), whether the visitor is a human or a bot, and how often these visitors access your website.
+
+### Fingerprint Metrics Reports
+
+The PhotonIQ Fingerprint service has several reports to show how often visitors return to your site, the type of devices that are accessing your site, and what kind of bots are accessing your site.
+
+### Top N Visitors
+
+The top N visitor reports the top n number of visitors that have returned to your website over a given date/time range.
+
+### Device Types
+
+The device types report returns all the different types of devices and how many of them accessed your website.
+
+### Visitor Traffic
+
+The visitor traffic report returns the number of good bots, bad bots, unknown bots, and humans who accessed your website. The report returns data in five-minute intervals over a date/time range of your choosing.
+
+## Best Practices
+
+### 1st Party vs 3rd Party Cookies
+
+The Photoniq Fingerprint service uses [third-party cookies](first-third-party-cookie) by default, which may affect the accuracy of the identification process. To ensure high levels of identification accuracy, it is recommended to use a customer subdomain or a cloud proxy (such as Akamai). By employing this approach, the cookies set by the Fingerprint service will be treated as [first-party cookies](first-third-party-cookie), which will enhance the identification precision.
+
+### Fingerprint Browser Agent on Page Load
+
+The JavaScript agent is loaded at page load time, and the function to get the visitor ID is run right away. For more information, click [here]().
```

Comment:
I need help to get this to point the the "page load" section of the getting started guide.

Reply:
You mean link to it? I will add a link.